docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md

```diff
@@ -117,4 +117,40 @@ The authentication mechanism for accessing blobs using yb-voyager is the same as
 Refer to [Sign in with Azure CLI](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli) for additional details on setting up your Azure blobs.
   {{% /tab %}}
 
-{{< /tabpane >}}
\ No newline at end of file
+{{< /tabpane >}}
+
+## End migration
+
+To end the migration, you need to clean up the export directory (export-dir), and Voyager state ( Voyager-related metadata) stored in the target database.
+
+Run the `yb-voyager end migration` command to perform the clean up, and to back up the schema, data, migration reports, and log files by providing the backup related flags (mandatory) as follows:
+
+```sh
+yb-voyager end migration --export-dir <EXPORT_DIR>
+        --backup-log-files <true, false, yes, no, 1, 0>
+        --backup-data-files <true, false, yes, no, 1, 0>
+        --backup-schema-files <true, false, yes, no, 1, 0>
+        --save-migration-reports <true, false, yes, no, 1, 0>
+```
+
+Note that after you end the migration, you will _not_ be able to continue further. If you want to backup the log file and import data status output for future reference, the command provides an argument as `--backup-dir`, using which you can pass the path of the directory where the backup content needs to be saved (based on what you choose to back up).
+
+Refer to [end migration](../../reference/end-migration/) for more details on the arguments.
+
+## Verify migration
+
+After the data import is complete, the automated part of the importing the data from files is considered complete. You should manually run validation queries on the target database to ensure that the data is correctly imported. A sample query to validate the databases can include checking the row count of each table.
+
+{{< warning title = "Caveat associated with rows reported by import data status" >}}
+
+Suppose you have the following scenario:
+
+- [import data file](../bulk-data-load/#import-data-files-from-the-local-disk) command fails.
+- To resolve this issue, you delete some of the rows from the split files.
+- After retrying, the import data to target command completes successfully.
+
+In this scenario, the [get data-migration-report](#get data-migration-report) command reports an incorrect imported row count because it doesn't take into account the deleted rows.
+
+For more details, refer to the GitHub issue [#360](https://github.com/yugabyte/yb-voyager/issues/360).
+
```

Comment:
change the order of End migration and Verify migration section, first Verify Migration and then End migration.